packages/rocketchat-api/server/v1/im.js

```diff
@@ -177,6 +177,68 @@ RocketChat.API.v1.addRoute(['dm.history', 'im.history'], { authRequired: true },
 	},
 });
 
+RocketChat.API.v1.addRoute(['dm.history.others', 'im.history.others'], { authRequired: true }, {
+	get() {
+		// const findResult = findDirectMessageRoom(this.requestParams(), this.user);
```

Comment:
Can you please remove this commented code?

Reply:
i just updated my branch and removed the comment.